src/types/List.js

```diff
@@ -0,0 +1,38 @@
+import { extend } from '../utils/adopter.js'
+import { subClassArray } from './ArrayPolyfill.js'
+
+const List = subClassArray('List', Array, function (arr) {
+  this.length = 0
+  this.push(...arr)
+})
+
+export default List
```

Comment:
Maybe there should be tests for this?

Reply:
Maybe there should be tests for the whole fx module ;)